### PR TITLE
Youtube player template and first video embed

### PR DESCRIPTION
**Pull request by johanandren**
and first video blog post

Screenshot 2019-12-03 at 15 57 41

No idea how to make that responsive/work on small screens though.

### CODE REVIEW

**Comment by johanandren** [comment]
On second thought, I'll spend some more time getting this responsive, hold off merging

File: blog/_posts/2019-12-03-akka-typed-actor-intro-video.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+layout: post
+title: Akka 2.6 Actor introduction video
+author: Johan Andrén
+short: A short Java live-coding introduction to the new actor APIs in Akka 2.6
+category: news
+tags: [videos]
+youtubeId: rIFqJxMJ1MM
+---
+
+{% include youtubePlayer.html id=page.youtubeId %}
```
Comment (patriknw) [inline blog/_posts/2019-12-03-akka-typed-actor-intro-video.md]: nice

File: blog/_posts/2019-12-03-akka-typed-actor-intro-video.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+A short video where we walk through the basics of the actor model writing an actor in Java with the new Actor APIs Akka 2.6
+
+For more getting started materials check out the [getting started section of the Akka documentation](https://doc.akka.io/docs/akka/current/typed/guide/introduction.html)
```
Comment (patriknw) [inline blog/_posts/2019-12-03-akka-typed-actor-intro-video.md]: or is it even better with https://akka.io/try-akka/ ?

**Comment by johanandren** [comment]
Small-screeney-mode: 

Screenshot 2019-12-03 at 16 23 49

File: blog/_posts/2019-12-03-akka-typed-actor-intro-video.md (hunk 2 above)
Comment (johanandren) [reply blog/_posts/2019-12-03-akka-typed-actor-intro-video.md]: I added both. Try-acka is more play around and getting started is more in-depth so both fits nicely I think.

File: _includes/youtubePlayer.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="tubeytube">
+  <iframe width="740" height="416" src="https://www.youtube.com/embed/{{ include.id }}" frameborder="0" allowfullscreen></iframe>
```
Comment (raboof) [inline _includes/youtubePlayer.html]:
```suggestion
  <iframe src="https://www.youtube.com/embed/{{ include.id }}" frameborder="0" allow="fullscreen"></iframe>
```

File: resources/stylesheets/sass/views/_blog-page.scss [hunk 4]
```diff
@@ -70,4 +70,19 @@
...
+		overflow:hidden;
+	}
+
+	.tubeytube iframe, .tubeytube object, .tubeytube embed {
+		position:absolute;
+		top:0;
+		left:0;
+		width:100%;
+		height:100%;
+	}
```
Comment (raboof) [inline resources/stylesheets/sass/views/_blog-page.scss]: :+1: nice

**Pull request merged**